s/frontend/components/common/five-stars/namaka-review.ts

```diff
@@ -0,0 +1,26 @@
+import {renderFiveStarRating, stateBuddy} from './render-five-star-display.js';
+import fiveStarRatingCss from './render-five-star-display.css.js';
+import {mixinStyles} from '../../../framework/mixins/mixin-styles.js';
+import {html, LitElement} from 'lit';
+
+@mixinStyles(fiveStarRatingCss)
+export class NamakaReview extends LitElement {
+	render() {
+
+		customElements.whenDefined('namaka-review').then(() => {
```

Comment:
what's this `whenDefined` call doing here?

Reply:
so i was getting some nulls wihout it, but need to figure it out this was just workaround